2000-10-30  Jakub Jelinek  <jakub@redhat.com>

	* cpplib.c (_cpp_parse_assertion): Don't crash if -A is given on the
	command line to cpp without opening paren.

--- gcc/cpplib.c.jj	Fri Oct 27 18:40:14 2000
+++ gcc/cpplib.c	Mon Oct 30 12:41:30 2000
@@ -1226,7 +1226,7 @@ _cpp_parse_assertion (pfile, answerp)
 	  if (token->type == CPP_EOF)
 	    goto lookup_node;
 	}
-      else if (d != &dtable[T_ASSERT])
+      else if (d != &dtable[T_ASSERT] && d)
 	{
 	  _cpp_push_token (pfile, token);
 	  goto lookup_node;
